src/main/java/techreborn/blockentity/storage/energy/HighVoltageSUBlockEntity.java

```diff
@@ -44,7 +44,7 @@
 	 * MFSU should store 4M Energy with 512 E/t I/O
 	 */
 	public HighVoltageSUBlockEntity(BlockPos pos, BlockState state) {
-		super(TRBlockEntities.HIGH_VOLTAGE_SU, pos, state, "HIGH_VOLTAGE_SU", 2, TRContent.Machine.HIGH_VOLTAGE_SU.block, RcEnergyTier.HIGH, 4_000_000);
+		super(TRBlockEntities.HIGH_VOLTAGE_SU, pos, state, "HIGH_VOLTAGE_SU", 2, TRContent.Machine.HIGH_VOLTAGE_SU.block, RcEnergyTier.HIGH, 100_000_000);
```

Comment:
MFSU should be 40M, not 100M. Reference: https://ftb.fandom.com/wiki/MFSU_(IndustrialCraft_2)

Reply:
I think it's cheesy that the mfsu stores almost 40% less energy than the sum of its parts but yeah 100M is a bit much